modules/mono/csharp_script.h

```diff
@@ -172,7 +172,7 @@ class CSharpInstance : public ScriptInstance {
 	bool base_ref;
 	bool ref_dying;
 
-	void _ml_call_reversed(MonoObject *p_mono_object, GDMonoClass *klass, const StringName &p_method, const Variant **p_args, int p_argcount);
+	void _ml_call_reversed(MonoObject *p_mono_object, GDMonoClass *p_klass, const StringName &p_method, const Variant **p_args, int p_argcount);
```

Comment:
This could be `p_class`, since that is not a reserved word.

Reply:
This method is being removed by #16205.